docker/development/README.md

```diff
@@ -0,0 +1,52 @@
+### Requirements
+1. [Docker](https://docs.docker.com/)  
+2. [Docker compose](https://docs.docker.com/compose/)
+
+### Environment file
+
+Copy and rename .env.dist from the app directory to this directory
+```
+cp .env.dist docker/development/.env
+cd docker/development
+mv .env.dist .env
+```
+
+### Manage your docker environment
+
+Make sure you are in directory
+`docker/development`  
+Run the following commands  
+```
+docker-compose up -d
+```  
+
+Now a Docker container named 'dca_php' is running in the background. You can execute now any command in this container. 
+For example `docker exec -it dca_php php --version` for retrieving the installed PHP version from the Docker image.
+
+To stop the Docker container from running, execute 
+```
+docker-compose stop
+```
+
+Top and remove the Docker container, execute
+```
+docker-compose down
+```
+
+If you wish to rebuild the Docker image from the Dockerfile, run `docker-compose --build`
+
+### Start development
+First, make sure you start with installing the software dependencies with Composer.  
+```
+docker exec -it dca_php composer install
```

Comment:
Het afleiden van de container naam kan hier achterwege gelaten worden: `docker-compose exec dev_dca composer install`

Reply:
Als ik geen `container_name` opgeef in de `docker-compose.yml` dan krijgt de container de naam development_dca_1

Reply:
Klopt, maar als je `exec` via docker-compose uitvoert kun je de naam van de service gebruiken en is de naam van je container niet meer relevant.

Reply:
Ah, dan wordt het command `docker exec -it <container_name> <command>` dus `docker-compose exec <service_name> <command>` met als kanttekening dat deze laatste command wel in de `docker/development` directory uitgevoerd moet worden :) Helder en is logischer mbt development!